src/Types/index.tsx

```diff
@@ -293,6 +293,8 @@ export enum IconFont {
   Disks = 'disks',
   DisksNav = 'disks-nav',
   Download = 'download',
+  DragToExpand = 'drag-to-expand',
+  DragToExpandWhite = 'drag-to-expand-white',
```

Comment:
All the icons can have different colors so having the color "white" in the icon name is problematic. If it's a solid or outline variant call them as such.